spec/features/allocate_feature_spec.rb

```diff
@@ -51,9 +51,8 @@
     visit prison_prisoner_staff_index_path('LEI', nomis_offender_id)
     find('#non-recommended-accordion-section-heading').click
 
-    # Amit is 6th on the non-recommended list
     within '#non-recommended-accordion-section' do
-      within 'tbody > tr:nth-child(6)' do
+      within row_containing 'Amit Muthu' do
         click_link 'Allocate'
       end
     end
```

Comment:
@ollietreend Does it still rely on Amit Muthu to always be there?

Reply:
@Katy600 Yeah, it does 😕 which isn't ideal.

I've just updated it to use our test POM "Moic Pom" instead. Thanks for the nudge to do this! 🤓 🙌🏻